updatecli/weekly.d/puppet-modules/firewall.yaml

```
@@ -0,0 +1,59 @@
+---
+name: Bump the Firewall Puppet Module
```

```suggestion
name: Bump the puppetlabs-firewall module
```
Nit, more no-brainer
A matter of taste, feel free do decline.